ui/DesignSystem/Component/SourceBrowser.svelte

```diff
@@ -1,86 +0,0 @@
-<script>
```

Comment:
Any specific reason why we can't keep it contained in a component? We'll have the same source browser functionality in orgs as well.

Reply:
I wanted to contain any data access under the screen hierarchy. In what context will source browsing done for orgs? Only projects have repos, right?

Reply:
Right, yeah it'll be only for projects. I remember there was a point where we had a distinct project landing page (which shows the readme and the source browser) and then a separate "Source" tab with only the source browsing. Right now we don't have this in the designs anymore, so maybe we won't need it.